expo/tests/gbe/test_create_volunteer.py

```diff
@@ -25,15 +25,17 @@ def setUp(self):
         self.performer = PersonaFactory()
         Conference.objects.all().delete()
         self.conference = ConferenceFactory(accepting_bids=True)
-        days = ConferenceDayFactory.create_batch(2, conference=self.conference)
+        days = ConferenceDayFactory.create_batch(3, conference=self.conference)
```

Comment:
Below in the get_volunteer_form() function I do some slicing for the available and unavailable windows. The intention was to use the third ConferenceDay as an unavailable window for when the form is being submitted.

```python
def get_volunteer_form(self, submit=False, invalid=False):
        form = {'profile': 1,
                'number_shifts': 2,
                'interests': ['VA0'],
                'available_windows': self.conference.windows().values_list(
                    'pk', flat=True)[0:2],
                'unavailable_windows': self.conference.windows().values_list(
                    'pk', flat=True)[2]
                }
        if submit:
            form['submit'] = True
        if invalid:
            del(form['number_shifts'])
        return form
```